sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/storage/GcsKmsKeyIT.java

```diff
@@ -66,14 +65,16 @@ public static void setup() {
   /**
```

Comment:
The `@BeforeClass` method shouldn't be needed since TestPipelineOptions should be automatically registered with PipelineOptionsFactory

Reply:
removed